managed/services/telemetry/telemetry_test.go

```diff
@@ -39,8 +38,29 @@ import (
 	"github.com/percona/pmm/managed/utils/testdb"
 )
 
+const (
+	envPGHostPort = "TEST_PG_HOST_PORT"
+	envQanDSN     = "TEST_QAN_DSN"
+	envVmDSN      = "TEST_VM_DSN"
+)
+
 func TestRunTelemetryService(t *testing.T) {
-	t.Skip()
+	pgHostPort := "127.0.0.1:5432"
+	pgHostPortFromEnv, ok := os.LookupEnv(envPGHostPort)
+	if ok {
+		pgHostPort = pgHostPortFromEnv
+	}
+	qanDSN := "tcp://localhost:9000?database=pmm&block_size=10000&pool_size="
```

Comment:
`block_size` & `pool_size` not available for "github.com/ClickHouse/clickhouse-go/v2"

Reply:
fixed.